### ISSUE TITLE
Coupon Bug

### ISSUE THREAD

**Issue by shreyans94** [woocommerce/woocommerce#17440]
Usage limit per user is set to 1 and Usage limit per coupon is unlimited, the coupon gives max limit errors.

Please see this video:

https://www.useloom.com/share/bf3a184d8bd042678779a3c026a6b6df

**Comment by Grauen** [comment]
Same here. There are some other issues about this topic. I've read this will be fixed with 3.2.2 but I cannot find the issue right now.

The coupon which causes the problem in my case is a 100% price reduction coupon. It is limited to 2 products in the cart and it is limited to one usage per customer. I updated it, adding a usage limit of 1000 but this didn't help.

Before I've sent this coupon to our customers, I tested it with an existing test-account and I was able to checkout properly, using the coupon. After this test, I added this coupon as a guest. The payment options where hidden from the checkout form and there was only free checkout available. I was like - "yay, it's working, no need to proceed testing here". 

Imagine our surprise, after I've sent this coupon to 500 users. It's a mess and we've received a lot of support request so far. I'm so glad I didn't sent this to all recipients - about 15k users. 

We now have many aborted orders with credit cart selected as payment option. I gues the customers didn't see the "limit reached"-error and then they just continued to the checkout process.  I hope you will release 3.2.2 very soon :).
